DICOM/src/Microsoft.Health.DeID.SharedLib.UnitTests/EncryptTests.cs

```diff
@@ -115,104 +123,103 @@ public static IEnumerable<object[]> GetInvalidCryptoGraphicTextDataToDecrypt()
-            var plainText = EncryptFunction.DecryptContentWithRSA(cipherText, PrivateKey);
+            var cipherText = encryptFunction.EncryptContentWithRSA(originalText);
+            var plainText = encryptFunction.DecryptContentWithRSA(cipherText);
             Assert.Equal(originalText, plainText == null ? null : plainText.Length == 0 ? string.Empty : Encoding.UTF8.GetString(plainText));
         }
 
         [Theory]
         [MemberData(nameof(GetBytesDataToEncrypt))]
         public void GivenAnOriginalBytes_WhenEncryptWithRSA_ResultShouldBeValidAndDecryptable(byte[] originalBytes)
         {
-            RSACryptoServiceProvider rsa = new RSACryptoServiceProvider();
-            var cipherText = EncryptFunction.EncryptContentWithRSA(originalBytes, rsa.ExportRSAPublicKey());
-            var plainText = EncryptFunction.DecryptContentWithRSA(cipherText, rsa.ExportRSAPrivateKey());
+            var cipherText = encryptFunction.EncryptContentWithRSA(originalBytes);
+            var plainText = encryptFunction.DecryptContentWithRSA(cipherText);
             Assert.Equal(originalBytes, plainText);
         }
 
         [Theory]
         [MemberData(nameof(GetStreamDataToEncrypt))]
         public void GivenAnOriginalStream_WhenEncryptWithRSA_ResultShouldBeValidAndDecryptable(Stream originalStream)
         {
-            RSACryptoServiceProvider rsa = new RSACryptoServiceProvider();
-            var cipherText = EncryptFunction.EncryptContentWithRSA(originalStream, rsa.ExportRSAPublicKey());
-            var plainText = EncryptFunction.DecryptContentWithRSA(cipherText, rsa.ExportRSAPrivateKey());
+            var cipherText = encryptFunction.EncryptContentWithRSA(originalStream);
+            var plainText = encryptFunction.DecryptContentWithRSA(cipherText);
             Assert.Equal(StreamToByte(originalStream), plainText);
         }
 
         [Theory]
         [MemberData(nameof(GetBytesDataToDecryptUsingRSA))]
         public void GivenAnEncryptedBytes_WhenDecryptUsingRSA_OriginalTextShouldBeReturned(byte[] cipherText, string originalText)
         {
-            var plainText = EncryptFunction.DecryptContentWithRSA(cipherText, PrivateKey);
+            var plainText = encryptFunction.DecryptContentWithRSA(cipherText);
             Assert.Equal(originalText, plainText == null ? null : plainText.Length == 0 ? string.Empty : Encoding.UTF8.GetString(plainText));
         }
 
         [Theory]
         [MemberData(nameof(GetStreamDataToDecryptUsingRSA))]
         public void GivenAnEncryptedStream_WhenDecryptUsingRSA_OriginalTextShouldBeReturned(Stream cipherText, string originalText)
         {
-            var plainText = EncryptFunction.DecryptContentWithRSA(cipherText, PrivateKey);
+            var plainText = encryptFunction.DecryptContentWithRSA(cipherText);
             Assert.Equal(originalText, plainText == null ? null : plainText.Length == 0 ? string.Empty : Encoding.UTF8.GetString(plainText));
         }
-
+        */
```

Comment:
Just curious why these tests are commented?

Reply:
I will add these tests in shared lib PR. Since these tests are RSA tests not effect DICOM.